app/frontend/src/components/ApplicationList.vue

```diff
@@ -52,18 +61,23 @@ export default {
   name: 'ApplicationList',
   data: () => ({
     // TEMP for testing skeletons
-    waiting: true
+    waiting: true,
+    clientExistsText: 'Available',
+    clientNotExistsText: 'Not Available'
   }),
   computed: {
     ...mapGetters('user', ['acronyms'])
   },
   methods: {
-    ...mapActions('user', ['getUserAcronyms'])
+    ...mapActions('user', ['getUserAcronyms', 'fillInAcronymClientStatus'])
   },
-  mounted() {
-    this.getUserAcronyms();
-    // TEMP for testing skeletons
-    setTimeout(() => (this.waiting = false), 1000);
+  async mounted() {
+    //TODO: consider moving getUserAcronyms to on login instead of this component
```

Comment:
If it's done on "login", that means this needs to be executed on the App.vue level or even higher up (main.js). There's benefits and drawbacks of making the call earlier in the lifecycle.

Reply:
Yes that's one of the discussion points for tomorrow.

Reply:
Will leave this and revisit with Jeremy after auth state changes.

Reply:
Will be using vuex lazy loading on view level to address on-demand loading.